src/components/ExhibitionCard/index.jsx

```diff
@@ -0,0 +1,50 @@
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+
+import styles from './styles.module.css';
+
+const ExhibitionCard = ({ id, title, shortDescription, imageId, imageUrl, status, galleryTitle, startAt, endAt }) => {
+  let imageSrc = `${process.env.PUBLIC_URL}/noImage.jpg`;
+  if (imageUrl) {
+    imageSrc = imageUrl;
+  } else if (imageId) {
+    imageSrc = `https://www.artic.edu/iiif/2/${imageId}/full/300,/0/default.jpg`;
+  }
+
+  return (
+    <div className={styles.container}>
+      <Link to="/exhibitions/:exhibitionsId" key={id}>
+        <h5 className={styles.title}>Title: {title}</h5>
+        <h5 className={styles.description}>Description: {shortDescription}</h5>
+        <img className={styles.imageId} src={imageSrc} alt={title} style={{ width: '300px', height: '200px' }} />
```

Comment:
We added this inline style temporarily just to make the image look good enough, but these CSS rules should be defined in the styles.module.css file. Additionally, the image should scale appropriately instead of hardcoding both its width and height; you may use the styling in the art work card image as a reference to implement this one.